app/assets/javascripts/app/controllers/ticket_zoom/meta.coffee

```diff
@@ -14,3 +16,23 @@ class App.TicketZoomMeta extends App.ControllerObserver
       isCustomer: ticket.currentView() is 'customer'
     )
     @renderPopovers()
+
+  copyTicketNumber: =>
```

Comment:
I the the same code 2 times. It would be great if copyTicketNumber function would be a generic function in app/assets/javascripts/app/controllers/_application_controller/_base.coffee  where you just need to pass a selector, the texts and maybe a callback if needed.

Reply:
Yeah, I had a feeling a comment like this would pop up, just was not sure where to best put this function. :smile:
Speaking of `_base.coffee`, is the existing copyToClipboard function there even used anywhere still? :thinking:

Reply:
Guess it's used potentially in addon packages.